pymc/bart/utils.py

```diff
@@ -147,13 +146,13 @@ def plot_dependence(
 
     rng = RandomState(seed=random_seed)
 
-    if isinstance(X, pd.DataFrame):
+    if hasattr(X, "columns") and hasattr(X, "values"):
         X_names = list(X.columns)
         X = X.values
     else:
         X_names = []
 
-    if isinstance(Y, pd.DataFrame):
+    if hasattr(Y, "name"):
```

Comment:
I think this was a bug. DataFrames do not have a `name` attribute.

Reply:
right, it was actually expecting a Series, but checking for a DataFrame.